### PR TITLE
os module, and executeCommand function

### PR DESCRIPTION
**Pull request by miniche**
This function will be use everywhere we need to execute an os command.
It's more flexible, if we want to replace the "subprocess.Popen" call in the future for example. Returns the result string.

Todo: 
- [x] Create the os module
- [x] Design the executeCommand function
- [x] Replace code everywhere is needed to use the new function instead of "subprocess" call

### CODE REVIEW

File: centralreport/cr/os.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    https://github.com/miniche/CentralReport/
+"""
+
+import shlex
+import subprocess
+
+
+def WIP_executeCommand(str_command):
+    """
+        This function executes an os command. Command can contains pipes, executed as subcommands.
```
Comment (Ninir) [inline centralreport/cr/os.py]:
``` python
str_command can contain pipes
```

'Find it weird to have "Command", which is implicitly related to the str_command parameter

File: centralreport/cr/os.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        If your command contains vars with a space, use " " to escape words (eg: "Volume Name")
+        If you want to use an double quote (") in your command, escape it with \\ (eg: \\":\\" return ":")
```
Comment (Ninir) [inline centralreport/cr/os.py]: Typo: `an`

File: centralreport/cr/os.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+        Returns the result of the command (string).
+    """
+
+    # Command can contains pipes. At each pipe, we execute a "subcommand".
```
Comment (Ninir) [inline centralreport/cr/os.py]: Typo: s to contain

File: centralreport/cr/os.py [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        print list_command
+        # If it's not the first occurence, stdin is the last command executed.
+        if 0 == i:
+            list_results.append(subprocess
+                                .Popen(list_command, stdout=subprocess.PIPE))
+        else:
+            list_results.append(subprocess
+                                .Popen(list_command, stdout=subprocess.PIPE, stdin=list_results[i - 1]
+                                .stdout))
+            # End the previous subprocess (http://docs.python.org/2/library/subprocess.html#replacing-shell-pipeline)
```
Comment (Ninir) [inline centralreport/cr/os.py]: Typo: Ends

**Comment by miniche** [comment]
Time to review! :innocent:

**Comment by Guimove** [comment]
Seems to be good for me

**Pull request merged**